<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>564</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>400</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="value_area">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="value_widget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>238</width>
         <height>544</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="value_layout">
        <item row="2" column="0">
         <widget class="QLabel" name="label_voltage">
          <property name="text">
           <string>Voltage</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spinbox_voltage"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_dt">
          <property name="text">
           <string>Time Step</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="spinbox_dt"/>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QPushButton" name="file_import_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Import Files</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QComboBox" name="file_list_box"/>
        </item>
       </layout>
       <zorder>label_voltage</zorder>
       <zorder>label_dt</zorder>
       <zorder>spinbox_voltage</zorder>
       <zorder>spinbox_dt</zorder>
       <zorder>label_voltage</zorder>
       <zorder>label_dt</zorder>
       <zorder>spinbox_voltage</zorder>
       <zorder>spinbox_dt</zorder>
       <zorder>file_import_button</zorder>
       <zorder>file_list_box</zorder>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="graph_layout"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
